setup.py

```diff
@@ -1,12 +1,5 @@
 import sys
-from distutils.core import setup
-
-if sys.version_info[:2] < (2, 5) or sys.version_info[0] > 2:
```

Comment:
If the Python version requirement is changed, please update the version check and classifiers, but don't remove the version check.  There's no reason to allow the package to be installed on known incompatible versions.